.tx/config

```diff
@@ -4,7 +4,23 @@ host = https://www.transifex.com/
 [OpenMRS.haiti-core-module]
 source_file = api/src/main/resources/messages.properties
 source_lang = en
-type = UNICODEPROPERTIES
```

Comment:
Did you intentionally remove the above line 7?

Reply:
No, that was an unintentional change. Would you like me to fix and recommit?